runtime/moonbase/tests/integration_test.rs

```diff
@@ -1908,7 +1908,7 @@ fn initial_gas_fee_is_correct() {
 			TransactionPaymentAsGasPrice::min_gas_price(),
 			(
 				2_500_000_000u128.into(),
-				Weight::from_parts(41_742_000u64, 0)
+				Weight::from_parts(59_217_000u64, 0)
```

Comment:
This is the read cost, we should use `<Runtime as frame_system>::DbWeight::get().read` or `moonbase_weights::db::rocksdb::constants::RocksDbWeight::get().read`.

Reply:
Addressed in https://github.com/moonbeam-foundation/moonbeam/pull/3459/commits/789f99c50158da070d3d9256a8e78b23316343eb